logictest/testdata/aggregate/aggregate

```diff
@@ -28,6 +28,17 @@ select count(value) as value_count group by labels.label2
 ----
 value2  6
 
+exec
+select avg(value) as value_avg group by labels.label2
+----
+value2  3
+
+exec
+select avg(value) as value_avg group by labels.label4
```

Comment:
I think we should also run this function in tests with multiple row groups (i.e. insert statements), I think we already had tests like this for the other functions too?

Reply:
Above we have two inserts, so multiple row groups, that should be what you are looking for, isn't it?

Reply:
I see, in that case how is this passing without an `exec unordered` argument? The output ordering shouldn't be deterministic I think.